lib/iris/util.py

```diff
@@ -1554,3 +1554,111 @@ def _meshgrid(*xi, **kwargs):
         if mxii.dtype != xii.dtype:
             mxi[i] = mxii.astype(xii.dtype)
     return mxi
+
+
+def find_discontiguities(cube, rel_tol=1e-5, abs_tol=1e-8):
+    """
+    Searches coord for discontiguities in the bounds array, returned as a
+    boolean array (True where discontiguities are present).
+
+    Args:
+
+    * cube (`iris.cube.Cube`):
+        The cube to be checked for discontinuities in its 'x' and 'y'
```

Comment:
Pernickety but...
Ideally I'd change `discontinuities` -> `discontiguities` as it is referred to in the first sentence of this doc string.

There are a few places in this PR where the word `discontinuous` is used, but others where `discontiguous` is used. It would be nice if we could remain consistent and my preference is for `discontiguous`

Reply:
@lbdreyer You're quite right.  Those are just ones I missed.  Thanks!